### ISSUE TITLE
code details

### ISSUE THREAD

**Issue by Wangpeiyi9979** [RaptorMai/online-continual-learning#17]
Thanks for your nice work. I have a small question:

in `experiments/run.py`, there exist two similar functions: `def multiple_run_tune(default_params, tune_params, save_path)` and `def multiple_run_tune_separate(default_params, tune_params, save_path)`.
In my opinion, both of them first tune the model hyperparameters on the $D_{cv}$ and then conduct online continual learning on $D_{ev}$. 
Could you tell me what is the difference between these two functions, and which function do you use to achieve the results in Table7 and Table9 in the paper?

Thanks.

**Comment by czjghost** [comment]
Hello, as far as I can see, "multiple_run_tune" uses the data of the first "params.num_val" tasks to "tune_hyper", and it returns the best parameters which has the best average accuracy over three runs. Pay attention to the following content! Then, from "params.num_val + 1"-th task, the function will train the model. But, for "multiple_run_tune_separate", if it enters "single_tune_train_val", it will run another command "data_continuum.reset_run()", and then make model train from the first task; if it enters "single_tune", it is the same as "multiple_run_tune". This is my all understanding about this code.

multiple_run_tune  是  取前params.num_val个task  的训练集和测试集进行 tune_hyper ，并且跑params.num_runs_val 取平均acc最高的为最优参数，然后接着从params.num_val + 1 个task 开始直接训练。
而对于 single_tune_train_val 是 是类似于multiple_run_tune  但是区别在于 data_continuum.reset_run()，也就是又从第一个任务开始训练和测试了，对于 single_tune 他如果online=True的时候，就我个人理解而言，和multiple_run_tune是一样

**Comment by czjghost** [comment]
For, recent paper about online class incremental learning, I suggest that you could only see  "multiple_run" and the holder "config_CVPR" in detail.
